### PR TITLE
Read electron counted data from HDF5 file

### PR DESCRIPTION
**Pull request by psavery**
Basically, you can now read the electron counted data from an
HDF5 file and then use it to create a stem image like so:
```python
counts = io.read_electron_counts('/path/to/file.h5')
stem_img = image.create_stem_images(counts, inner_radius, outer_radius)[0]
```

You can also now create your own python class that has the attributes
`data`, `scan_dimensions`, and `frame_dimensions`, and you can pass
that into `image.create_stem_images()` to be used as electron counted data.

Fixes: #140

### CODE REVIEW

**Comment by cjh1** [comment]
@psavery As part of this lets update save_electron_counts, so that it will also accept a ElectronCountData object ( so your not force to pass in the dimensions as well).

File: python/stempy/io/__init__.py [hunk 1]
```diff
@@ -162,6 +162,31 @@ def save_electron_counts(path, events, scan_dimensions, frame_dimensions=None):
 
         frames[...] = events
 
+def read_electron_counts(path):
```
Comment (cjh1) [inline python/stempy/io/__init__.py]: I think this should be `load_electron_counts` ( as in save/load )

Comment (psavery) [reply python/stempy/io/__init__.py]: Done

**Comment by psavery** [comment]
@cjh1 Done! This is ready for review again.

**Pull request merged**